crypto/packet/src/packet.rs

```diff
@@ -30,6 +30,8 @@ pub const PACKET_LENGTH: usize = packet_length::<CurrentSphinxSuite>(INTERMEDIAT
 /// Tag used to separate padding from data
 const PADDING_TAG: &[u8] = b"HOPR";
 
+pub const USABLE_PAYLOAD_SIZE: usize = PAYLOAD_SIZE - PADDING_TAG.len();
```

Comment:
Let's make `PAYLOAD_SIZE` non-public constant then.

Reply:
It is public, because the session needs to know that for unwrapping.